gradle/build-scan.gradle

```diff
@@ -3,38 +3,28 @@ import org.elasticsearch.gradle.OS
 buildScan {
     URL jenkinsUrl = System.getenv('JENKINS_URL') ? new URL(System.getenv('JENKINS_URL')) : null
     String buildNumber = System.getenv('BUILD_NUMBER')
+    String buildUrl = System.getenv('BUILD_URL')
     String jobName =  System.getenv('JOB_NAME')
 
     tag OS.current().name()
 
-    // Accept Gradle ToS when project property org.elasticsearch.acceptScanTOS=true or this is an Elastic CI build
-    if (jenkinsUrl?.host?.endsWith('elastic.co') || Boolean.valueOf(project.findProperty('org.elasticsearch.acceptScanTOS') ?: "false")) {
-        termsOfServiceUrl = 'https://gradle.com/terms-of-service'
-        termsOfServiceAgree = 'yes'
+    if (jenkinsUrl?.host?.endsWith('elastic.co')) {
+        publishAlways()
+        buildScan.server = 'https://gradle-enterprise.elastic.co'
     }
 
     // Jenkins-specific build scan metadata
     if (jenkinsUrl) {
         tag 'CI'
-        tag System.getenv('JOB_NAME')
-        link 'Jenkins Build', System.getenv('BUILD_URL')
-        link 'Additional Logs',
-                "https://console.cloud.google.com/storage/elasticsearch-ci-artifacts/jobs/${jobName}/build/${buildNumber}.tar.bz2"
-        System.getenv('NODE_LABELS').split(' ').each {
+        tag jobName
```

Comment:
I wonder if we should add the build number as a custom value as well, that way we could search for all scans for a given job? Or even the URL itself as a custom value would serve the same purpose, as right now you can search on links.